src/aliceVision/sfmData/View.hpp

```diff
@@ -200,7 +213,7 @@ class View
    */
   bool isPoseIndependant() const
```

Comment:
@fabiencastan I can fix the typo in this one as well but I'd rather do it before merging to avoid introducing too much noise, as there are quite few instances to change

Reply:
do you mean after merging?

Reply:
no, just before the PR is ready to be merged, i can make the refactoring. Just to avoid noise in the review